### DISCUSSION TITLE
How to easily include images

### DISCUSSION THREAD

**Discussion by reagle** [freeplane/freeplane#342] [Help]
I'd like to easily pull/paste an image into Freeplane.

I know there's a plugin that allows you to include image URLs, but they're not easy to adjust for size and such.

I know I can also drag an image into Freeplane natively, and have those stored on my file system at `~/scratch-pad_files`. Can we specify where we'd like this folder to be? Or, could there be a `media` folder within the folder of the target mindmap where such images go?

**Comment by sifran-github** [comment]
https://www.youtube.com/watch?v=EE3VGoZi0s8

this is an old video but its still relevant, and will answer some of your questions, especially about resizing.

**Comment by reagle** [comment]
Thank you, that is a useful video. Unfortunately, the first two methods aren't great and the extension is kind of broke for me. I can't get the "open folder" or "paste image from clipboard" features to work; and if it did, it'd still be great to be able to define a media folder, with an absolute path or relative path to a mindmap, for keeping the images.
